concent-builder/files/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,76 @@
+properties([[$class:
+'GithubProjectProperty',
+    displayName: '',
+    projectUrlStr: 'https://github.com/bartoszbetka/concent-deployment.git/'],
```

Comment:
The URL should come from our YAML config that contains repository addresses.

Reply:
The same https://github.com/golemfactory/concent-deployment/pull/284#discussion_r249460766